src/target/riscv/riscv.c

```diff
@@ -4598,52 +4747,69 @@ COMMAND_HANDLER(riscv_set_autofence)
 	return ERROR_COMMAND_SYNTAX_ERROR;
 }
 
-COMMAND_HANDLER(riscv_set_ebreakm)
+COMMAND_HELPER(ebreakx_deprecation_helper, enum riscv_ebreak_ctl mode)
 {
-	struct target *target = get_current_target(CMD_CTX);
-	RISCV_INFO(r);
-
+	struct target * const target = get_current_target(CMD_CTX);
+	struct riscv_private_config * const pc = riscv_private_config(target);
```

Comment:
Please, choose a different identifier than `pc` here (and in all other places). The reason is that `pc` commonly stands for program counter.

I noticed that some other targets do use `pc` as an ID for `private_config`, but it would IMO be better to avoid this confusion.

For example:

```suggestion
	struct riscv_private_config * const config = riscv_private_config(target);
```

Reply:
Addressed.